### ISSUE TITLE
[FEATURE REQUEST] Allow TestsNotExecuted to show as a non-fatal warning

### ISSUE THREAD

**Issue by PeterConijn** [HodorNV/ALOps#741]
**Is your feature request related to a problem? Please describe.**
In order to speed up our automated tests, we have split the tests into separate batches as per [this blog](https://www.waldo.be/2023/12/28/speed-up-your-automated-tests-without-really-doing-anything/). The author is awesome.

The problem is that you never quite know how many batches you get. I have enhanced the install code in the blog a little so that when I publish the test app, it automatically generates however many batches are needed. However, I cannot find a way to do this in the yml (this is sort of related to #725) so I need to hard-code it there. I am looking to keep this slightly better maintainable.

**Describe the solution you'd like**
Since automatic detection is difficult, I was thinking to add one more testsuite batch than we currently have. However, this would currently fail the build, since "No Tests Were Executed" counts as a failure. 

Is it possible to somehow create a setting to show this as informational and not have it list the build as failed or partially successful, but as successful? That was I can monitor and as soon as the message no longer appears (which means our tests now have expanded to this batch, I can update the pipeline with the next (empty) batch. 

It's not ideal, but better than having tests that are not run in the pipeline because I was too late in appending the test batches.

**Describe alternatives you've considered**
Right now, we manually publish the tests locally and monitor the batches, but with the number of devs and human nature, the risk of forgetting it is high.

**Additional context**
Test suite batches
![image](https://github.com/HodorNV/ALOps/assets/47315514/d8d58f02-c3d2-4af1-9b59-4e1283087b61)

Yml snippet to run the tests
![image](https://github.com/HodorNV/ALOps/assets/47315514/6d371203-79db-4f84-b328-6df77f33f5e1)

**Comment by waldo1001** [comment]
We added a setting "allow_zero_tests".

**Comment by waldo1001** [comment]
Can you try:

```yaml
- task: Hodor.hodor-alops.alopsapptest.ALOpsAppTest@1
  displayName: 'ALOps App Test'
  inputs:
    allow_zero_tests: true
```

**Comment by waldo1001** [comment]
Released in v1.465
